### DISCUSSION TITLE
lsv - yet another ls alternative

### DISCUSSION THREAD

**Discussion by mike-ward** [vlang/v#21702] [Announcements]
I've been contributing as of late to the Coreutils project. I started implementing a `ls` program but it kind of got away from me. I blame `V` for being way too much fun for this 😄.  I also feel maybe I strayed a bit too far from the Coreutils goals of GNU equivalents, so I'm publishing it [here](https://github.com/mike-ward/lsv)

Obviously, this is my interpretation of what a good `ls` program should be. Try it out if you have time and send me feedback/suggestions.

**Comment by Delta456** [comment]
Hi @mike-ward! `lsv` looks awesome. It feels like I am using `exa` but in V. I would love to get this added in https://github.com/vlang/awesome-v

**Reply by mike-ward** [reply]
Sent a pull request

**Comment by mike-ward** [comment]
Added table format (with header and blocked options and misspellings 🫨 ). More rabbit holes!. 

![image](https://github.com/vlang/v/assets/567927/0ab08d99-882d-4eec-b6b2-2447680ea890)
